force_wfmanager/ui/setup/mco/mco_parameter_view.py

```diff
@@ -0,0 +1,254 @@
+from traits.api import (
```

Comment:
This file has been refactored from `mco_parameter_model_view`, though Git recognises it as a new file

Reply:
Ah, this file contains both `KPISpecificationView` and `KPISpecificationModelView` classes, which is confusing I agree. In #313 this has been inadvertently resolved (along with the MCO parameters).